schemas/documents/registry/homePage.js

```diff
@@ -17,7 +17,7 @@ export default {
     },
     {
       name: 'projectsSection',
-      type: 'titleCustomBody',
+      type: 'homePageProjectsSection',
```

Comment:
just a side note that this is breaking and leads to the following error on https://dev.app.regen.network/ and all deploy previews that do not include your code from https://github.com/regen-network/regen-web/pull/2191 until this is merged
![image](https://github.com/regen-network/regen-sanity/assets/3149358/9daf5f47-d6ba-4bad-afef-953a1cc3d46e)


so it might be best to make use of sanity graphql tagged endpoints: https://www.sanity.io/docs/graphql#e2e900be2233

Reply:
I wasn't aware we could create additional tags like that, will make more use of that in the future if needed.

Reply:
I'll create an issue to add a note in the README about that